libs/appstore/src/portal/zomes/portal-zome-client.ts

```diff
@@ -49,10 +49,10 @@ export class PortalZomeClient extends ZomeClient {
         const availableHost = await Promise.any(
           hosts.map(async (hostEntryEntity) => {
             const hostPubKey = hostEntryEntity.content.author;
-            // console.log(
-            //   '@getAvailableHostForZomeFunction: trying to ping host: ',
-            //   encodeHashToBase64(hostPubKey),
-            // );
+            console.log(
+              '@getAvailableHostForZomeFunction: trying to ping host: ',
+              encodeHashToBase64(hostPubKey),
+            );
```

Comment:
It might be beneficial to implement structured logging, given the structured nature of the current console.log statements. This could enhance log clarity and make them more consistent and easier to parse. Somethign for later though, willing to support on this if you think it will be useful.

Reply:
Hmm yes, sounds interesting.

Reply:
Cool, I'll add this to the board so we can keep track of it